### DISCUSSION TITLE
How to convert FT.SEARCH Command to Redisearch search operation

### DISCUSSION THREAD

**Discussion by wazini-john** [danitseitlin/redis-modules-sdk-ts#163] [Q&A] [answered]
Hello
Would want to use the library to do some searching on hash. The hash is to run on a function with changing parameters which will in turn determine the results from the index. How do I convert it to a `await redisClient.search(...` operation

`FT.SEARCH testIndex '@type:{foobaz} -@baz:{NIL} @level:[0 +inf] @availableFoo:{testFoo} @cost:[0 +inf] @geoPoint:[136.754467 1.374688 1.5 km]' LIMIT 0 100`

**Comment by danitseitlin** [comment, marked as answer]
@wazini-john Please breakdown of each part of the command, you can also use typescript for better typing experiences. makes it easier for you

**Reply by wazini-john** [reply]
I have been able to do that... Thank you.

What do you advise 
1. Indexing the longitude and latitude 
2. Adding them as geoFilters
